api/src/bigquerytest/java/org/pmiops/workbench/api/CohortReviewControllerBQTest.java

```diff
@@ -166,27 +164,27 @@ public Clock clock() {
   private DbCdrVersion cdrVersion;
   private DbWorkspace workspace;
 
-  @Autowired private CohortReviewController controller;
+  @SpyBean private CohortReviewController controller;
```

Comment:
Good question. I reverted them back to Autowired. I don't think autowired is supposed to be used in a test. I would suspect that autowired is going to plug in actual component code rather than mocking. In that case, SpyBean also uses the actual functionality instead of MockBean where it relies on mocks. Since Autowired works for now, I will leave that alone, and we can address this later if we would like to.